updating lyrion music server fails

### ✅ Have you read and understood the above guidelines?

yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

lyrionmusicserver

### 📂 What was the exact command used to execute the script?

update

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 13

### 📈 Which Proxmox version are you on?

9.1.7

### 📝 Provide a clear and concise description of the issue.

the service seemingly gets renamed from lyrionmusicserver to lyrion. It tried to rename is manually after the update since a rename is only possible after the update due to the script. Is there a naming error?

### 🔄 Steps to reproduce the issue.

run update

### ❌ Paste the full error output (if available).

Note, selecting 'lyrionmusicserver' instead of '/tmp/lyrionmusicserver_9.1.0_amd64.deb'
Upgrading:
  lyrionmusicserver

Installing dependencies:
  psmisc

Summary:
  Upgrading: 1, Installing: 1, Removing: 0, Not Upgrading: 0
  Download size: 267 kB / 22.2 MB
  Freed space: 16.4 MB

Get:1 /tmp/lyrionmusicserver_9.1.0_amd64.deb lyrionmusicserver all 9.1.0 [21.9 MB]
Get:2 http://deb.debian.org/debian trixie/main amd64 psmisc amd64 23.7-2 [267 kB]
Fetched 267 kB in 0s (1,045 kB/s)           
apt-listchanges: Reading changelogs...
Selecting previously unselected package psmisc.
(Reading database ... 24723 files and directories currently installed.)
Preparing to unpack .../psmisc_23.7-2_amd64.deb ...
Unpacking psmisc (23.7-2) ...
Preparing to unpack .../lyrionmusicserver_9.1.0_amd64.deb ...
Unpacking lyrionmusicserver (9.1.0) over (9.0.3) ...
Setting up psmisc (23.7-2) ...
Setting up lyrionmusicserver (9.1.0) ...
Installing new version of config file /etc/squeezeboxserver/convert.conf ...
Installing new version of config file /etc/squeezeboxserver/modules.conf ...
Installing new version of config file /etc/squeezeboxserver/types.conf ...
audio
Processing triggers for man-db (2.13.1-1) ...
Failed to restart lyrion.service: Unit lyrion.service not found.
  ✖️   in line 40: exit code 5 (curl: Could not resolve proxy): while executing command systemctl restart lyrion

  --- Last 20 lines of log ---
[2026-04-13 23:01:27] [INFO] Updating Lyrion Music Server to 9.1.0
[2026-04-13 23:01:55] [ERROR] in line 40: exit code 5 (curl: Could not resolve proxy): while executing command systemctl restart lyrion

### 🖼️ Additional context (optional).

_No response_

--- a/ct/lyrionmusicserver.sh
+++ b/ct/lyrionmusicserver.sh
@@ -30,16 +30,16 @@ function update_script() {
     exit
   fi
 
-  DEB_URL=$(curl -s 'https://lyrion.org/getting-started/' | grep -oP '<a\s[^>]*href="\K[^"]*amd64\.deb(?="[^>]*>)' | head -n 1)
+  DEB_URL=$(curl_with_retry 'https://lyrion.org/getting-started/' | grep -oP '<a\s[^>]*href="\K[^"]*amd64\.deb(?="[^>]*>)' | head -n 1)
   RELEASE=$(echo "$DEB_URL" | grep -oP 'lyrionmusicserver_\K[0-9.]+(?=_amd64\.deb)')
   DEB_FILE="/tmp/lyrionmusicserver_${RELEASE}_amd64.deb"
   if [[ ! -f /opt/lyrion_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/lyrion_version.txt)" ]]; then
     msg_info "Updating $APP to ${RELEASE}"
-    curl -fsSL -o "$DEB_FILE" "$DEB_URL"
+    curl_with_retry "$DEB_URL" "$DEB_FILE"
     $STD apt install "$DEB_FILE" -y
-    systemctl restart lyrion
-    $STD rm -f "$DEB_FILE"
-    echo "${RELEASE}" >/opt/${APP}_version.txt
+    systemctl restart lyrionmusicserver
+    rm -f "$DEB_FILE"
+    echo "${RELEASE}" >/opt/lyrion_version.txt
     msg_ok "Updated $APP to ${RELEASE}"
     msg_ok "Updated successfully!"
   else

--- a/install/lyrionmusicserver-install.sh
+++ b/install/lyrionmusicserver-install.sh
@@ -14,10 +14,10 @@ network_check
 update_os
 
 msg_info "Setup Lyrion Music Server"
-DEB_URL=$(curl -fsSL 'https://lyrion.org/getting-started/' | grep -oP '<a\s[^>]*href="\K[^"]*amd64\.deb(?="[^>]*>)' | head -n 1)
+DEB_URL=$(curl_with_retry 'https://lyrion.org/getting-started/' | grep -oP '<a\s[^>]*href="\K[^"]*amd64\.deb(?="[^>]*>)' | head -n 1)
 RELEASE=$(echo "$DEB_URL" | grep -oP 'lyrionmusicserver_\K[0-9.]+(?=_amd64\.deb)')
 DEB_FILE="/tmp/lyrionmusicserver_${RELEASE}_amd64.deb"
-curl -fsSL -o "$DEB_FILE" "$DEB_URL"
+curl_with_retry "$DEB_URL" "$DEB_FILE"
 $STD apt install "$DEB_FILE" -y
 rm -f "$DEB_FILE"
 echo "${RELEASE}" >"/opt/lyrion_version.txt"

